Doxyfile

```diff
@@ -686,7 +686,7 @@ INPUT_ENCODING         = UTF-8
 # *.hxx *.hpp *.h++ *.idl *.odl *.cs *.php *.php3 *.inc *.m *.mm *.dox *.py
 # *.f90 *.f *.for *.vhd *.vhdl
 
-FILE_PATTERNS          =
+# FILE_PATTERNS          =
```

Comment:
Remove if not used.

Reply:
Disagree here. It's config template with full description, it's expected to be here